### PR TITLE
xdg-desktop-portal-gnome: update to 43.0.

### PR DESCRIPTION
**Pull request by oreo639**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/xdg-desktop-portal-gnome/template [hunk 1]
```diff
@@ -15,4 +15,4 @@ license="LGPL-2.1-or-later"
 homepage="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome"
 changelog="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/gnome-42/NEWS"
```
Comment (classabbyamp) [inline srcpkgs/xdg-desktop-portal-gnome/template:16]:
```suggestion
changelog="https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/raw/main/NEWS"
```

Comment (classabbyamp) [reply srcpkgs/xdg-desktop-portal-gnome/template]: the default branch is main, not master, and the changelog should be pulled from that branch, not the branch associated with a certain release

Comment (oreo639) [reply srcpkgs/xdg-desktop-portal-gnome/template]: > the default branch is main, not master

Thanks.

> the changelog should be pulled from that branch, not the branch associated with a certain release

It's commented out, the changlogs for specific branches are stored in that branch, when xdg-desktop-portal's main branch is gnome 44 and patches need to be backported for gnome 43, the changelogs for those patch releases won't appear in main usually.

Hence why you don't see any change information for 42.1, 42.2, and 42.3.

Comment (classabbyamp) [reply srcpkgs/xdg-desktop-portal-gnome/template]: ok

**Pull request merged**